### PR TITLE
implement `Paths` and make `Profile` a `Clone`

### PR DESCRIPTION
**Pull request by xphoniex**

### CODE REVIEW

File: radicle/src/profile.rs [hunk 1]
```diff
@@ -137,3 +142,18 @@ pub fn home() -> Result<PathBuf, io::Error> {
         ))
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct Paths<'a> {
+    home: &'a PathBuf,
```
Comment (cloudhead) [inline radicle/src/profile.rs:148]: Would be better to store a `&'a Path` here

Comment (xphoniex) [reply radicle/src/profile.rs]: fixed

**Comment by cloudhead** [comment]
I'm not sure what's up with the lints.. it seems like you're running clippy from 1.65.. Perhaps we can rebase this once the 1.65 branch is merged.

**Comment by xphoniex** [comment]
> I'm not sure what's up with the lints.. it seems like you're running clippy from 1.65.. Perhaps we can rebase this once the 1.65 branch is merged.

Yup, and as I was browsing other PRs, I realized you'd addressed these lint issues in https://github.com/radicle-dev/heartwood/pull/81.

Anyway rebased it and omitted the lints commit.

**Review by FintanH — APPROVED** [review]

**Pull request merged**